src/core/configuration/DefaultConfig.ts

```
@@ -529,55 +533,44 @@
       speed *= this.falloutDefenseModifier(falloutRatio);
     }
 
-    if (attacker.isPlayer() && defender.isPlayer()) {
-      if (
-        attacker.type() === PlayerType.Human &&
-        defender.type() === PlayerType.Bot
-      ) {
+    if (attacker.isPlayer() && defenderIsPlayer) {
+      if (attackerType == PlayerType.Human && defenderType == PlayerType.Bot) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)